server/src/testFixtures/java/io/crate/testing/LogicalPlanAssert.java

```diff
@@ -24,22 +24,36 @@
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 
 import org.assertj.core.api.AbstractAssert;
+import org.jetbrains.annotations.Nullable;
 
 import io.crate.planner.operators.LogicalPlan;
 import io.crate.planner.operators.PrintContext;
+import io.crate.planner.optimizer.costs.PlanStats;
 
 public class LogicalPlanAssert extends AbstractAssert<LogicalPlanAssert, LogicalPlan> {
 
+    @Nullable
+    private final PlanStats planStats;
+
     protected LogicalPlanAssert(LogicalPlan actual) {
+        this(actual, null);
+    }
+
+    protected LogicalPlanAssert(LogicalPlan actual, @Nullable PlanStats planStats) {
```

Comment:
The existing ctor has only 1 usage, maybe get rid of it instead of adding an overload?

Reply:
it's mandatory to override the default ctor of `AbstractAssert`.